packages/chakra-ui/src/components/buttons/clone/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import {
+    useCan,
+    useNavigation,
+    useTranslate,
+    useResource,
+    useRouterContext,
+} from "@pankod/refine-core";
+import {
+    RefineCloneButtonProps,
+    RefineButtonTestIds,
+} from "@pankod/refine-ui-types";
+import { Button, ButtonProps, IconButton } from "@chakra-ui/react";
+import { IconSquarePlus, TablerIconProps } from "@tabler/icons";
+
+export type CloneButtonProps = RefineCloneButtonProps<
+    ButtonProps,
+    {
+        svgIconProps?: TablerIconProps;
+    }
+>;
+
+/**
+ * `<CloneButton>` uses Mantine {@link https://chakra-ui.com/docs/components/button `<Button> component`}.
+ * It uses the {@link https://refine.dev/docs/core/hooks/navigation/useNavigation#clone `clone`} method from {@link https://refine.dev/docs/core/hooks/navigation/useNavigation useNavigation} under the hood.
+ * It can be useful when redirecting the app to the create page with the record id route of resource.
+ *
+ * @see {@link https://refine.dev/docs/ui-frameworks/chakra-ui/components/buttons/clone-button} for more details.
+ *
+ */
+export const CloneButton: React.FC<CloneButtonProps> = ({
+    resourceNameOrRouteName,
+    recordItemId,
+    hideText = false,
+    ignoreAccessControlProvider = false,
+    svgIconProps,
+    children,
+    onClick,
+    ...rest
+}) => {
+    const { resourceName, resource, id } = useResource({
+        resourceNameOrRouteName,
+        recordItemId,
+    });
+
+    const { cloneUrl: generateCloneUrl } = useNavigation();
+    const { Link } = useRouterContext();
+
+    const translate = useTranslate();
+
+    const { data } = useCan({
+        resource: resourceName,
+        action: "create",
+        params: { id, resource },
+        queryOptions: {
+            enabled: !ignoreAccessControlProvider,
+        },
+    });
+
+    const disabledTitle = () => {
+        if (data?.can) return "";
+        else if (data?.reason) return data.reason;
+        else
+            return translate(
+                "buttons.notAccessTitle",
+                "You don't have permission to access",
+            );
+    };
+
+    const cloneUrl = generateCloneUrl(resource.route!, id!);
+
+    return (
+        <Link
+            to={cloneUrl}
+            replace={false}
+            onClick={(e: React.PointerEvent<HTMLButtonElement>) => {
+                if (onClick) {
+                    e.preventDefault();
+                    onClick(e);
+                }
+            }}
+        >
+            {hideText ? (
+                <IconButton
+                    variant="outline"
+                    aria-label={translate("buttons.clone", "Clone")}
+                    title={disabledTitle()}
+                    data-testid={RefineButtonTestIds.CloneButton}
+                    {...rest}
+                >
+                    <IconSquarePlus size={18} {...svgIconProps} />
+                </IconButton>
+            ) : (
+                <Button
+                    size="sm"
```

Comment:
```suggestion
                    size="sm"
```
Could we use a global config/theme for this?